app/models/comment.rb

```diff
@@ -1,2 +1,9 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  after_save :update_comment_counter
+
+  def update_comment_counter
+    post.update(comments_counter: post.comments.all.length)
+  end
```

Comment:
- [ ] [Optional] Same here, It would be better professionally if you made the `update_comment_counter` a private method.